brevo/assets/monitors/brevo_high_bounce_rate_by_subject.json

```diff
@@ -0,0 +1,90 @@
+{
+	"version": 2,
+	"created_at": "2024-09-11",
+	"last_updated_at": "2024-09-11",
+	"title": "[Brevo] High Bounce Rate by Subject",
+	"description": "Monitors high bounce rates by subject.",
```

Comment:
Same here, can you update the title and description with the suggestions from our best practices guidelines: https://docs.datadoghq.com/developers/integrations/create-an-integration-recommended-monitor/#description